### PR TITLE
Minor Fixes on Bareos Create Cluster

### PR DESCRIPTION
**Pull request by patil-pratik-87**
Before :- 
Screenshot 2022-04-20 at 10 41 17 PM

After :-
 Screenshot 2022-04-21 at 10 39 45 AM

### CODE REVIEW

**Comment by jruddell** [comment]
worker nodes are not required on a single master cluster, so we shouldn't be requiring validation on this

**Comment by jruddell** [comment]
Please remove the validation changes you did. We have a validation system that checks for things. This here isn't using that. The form validation should be fine for now

File: src/app/core/elements/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -183,6 +183,10 @@ const Tooltip = styled(
       opacity: 1,
       transform: getMenuTransform(1)({ vertAlign, horizAlign }),
     },
+    '& *': {
```
Comment (jruddell) [inline src/app/core/elements/tooltip/Tooltip.tsx:186]: Generally styling this way is not efficient. Can we update the children to be styled correctly instead?

**Review by jruddell — CHANGES_REQUESTED** [review]
Let's clean this up some, please remove the validation stuff you did. I'll show you how this stuff works on a call later. 

Styling with a * selector should be avoided on highly used components because of the excess css styling in each instance

**Comment by patil-pratik-87** [comment]
> Please remove the validation changes you did. We have a validation system that checks for things. This here isn't using that. The form validation should be fine for now

done

**Comment by patil-pratik-87** [comment]
> Please remove the validation changes you did. We have a validation system that checks for things. This here isn't using that. The form validation should be fine for now

Yeah, we ve fixed the form validation and used that now :)

File: src/app/core/elements/tooltip/Tooltip.tsx (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/core/elements/tooltip/Tooltip.tsx]: true, used more specific classes

**Review by jruddell — APPROVED** [review]

**Pull request merged**